crates/holochain/src/core/workflow/countersigning_workflow/complete.rs

```diff
@@ -185,10 +174,8 @@ pub(crate) async fn inner_countersigning_session_complete(
         }
     }
 
-    publish_trigger.trigger(&"publish countersigning_success");
-
     tracing::info!(
-        "Countersigning session complete for agent {:?} in approximately {}ms",
+        "Countersigning session complete for agent {:?} in approximately {} ms",
```

Comment:
That is how I'd normally expect units to be presented, `1s` or `5m` rather than `120 ms`. If it were written in full, then yes a space, so `5 minutes`.

Reply:
True, that's a common way. That's not how [it's standardized](https://www.nist.gov/pml/owm/writing-si-metric-system-units), but it's become commonplace in the online world.

Reply:
Oh interesting, I hadn't realized that, standard is good, stick with standard :)